src/main/java/net/glowstone/GlowServer.java

```diff
@@ -122,8 +122,8 @@
 import net.glowstone.util.GlowHelpMap;
 import net.glowstone.util.GlowServerIcon;
 import net.glowstone.util.GlowUnsafeValues;
-import net.glowstone.util.Library;
-import net.glowstone.util.LibraryManager;
+import net.glowstone.library.Library;
+import net.glowstone.library.LibraryManager;
```

Comment:
Sorry that I wasn't clear, I meant net.glowstone.util.library, but I'm fine with this too. It's really up to you with which you feel is better.